content/finalist/Voomp/index.md

```diff
@@ -0,0 +1,66 @@
+---
+type: "finalist"
+title: Team Voomp
+name: "Voomp"
+description: "We at Voomp believe technology enhances every aspect of life and aim make it accessible to everyone."
+Team-Quote: "Driving forward with technology"
+Application_Type: "Mobile Application"
+---
+
+We at Voomp believe technology enhances every aspect of life and aim make it accessible to everyone.
+
+---
+
+# Project's Name
```

Comment:
mention project's Name is place of project's Name section

Reply:
Resolved